<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#widget{ background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0.0115909, y2:0.034, stop:0 rgba(177, 203, 228, 255), stop:0.806818 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255))}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 18pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>Animal Shelter </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 18pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="EditEnterMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:reflect, x1:0.926, y1:0.926136, x2:1, y2:1, stop:0 rgba(255, 170, 127, 255), stop:1 rgba(255, 206, 181, 255))</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>500</y>
      <width>20</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>16</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
   <widget class="QLineEdit" name="EditEnterId">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Input animal's identification number.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>20</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 38pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>Edit Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 18pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>Animal Id:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditEnterConfirm">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>560</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add New Animal To Database.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:reflect, x1:0.926, y1:0.92, x2:1, y2:1, stop:0 rgba(145, 255, 145, 255), stop:1 rgba(0, 223, 0, 255))</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
